addons/payment_transfer/data/transfer.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
-    <data noupdate="1">
+    <data>
```

Comment:
That means that at every update, the custom messages on the transfer acquirer will be reset to the default values ... does not seem a good idea to me :) .